### PR TITLE
Add limit for maximum of targets per user

### PR DESCRIPTION
**Pull request by MathiasLantean**
Common users just can add as many targets as it is defined in the project settings.

### CODE REVIEW

File: target/serializers.py [hunk 1]
```diff
@@ -10,9 +11,19 @@ class Meta:
         read_only_fields = ['user']
         geo_field = 'location'
 
+    def check_max_num_of_targets(self, user):
+        num_current_targets = Target.objects.filter(user=user).count()
```
Comment (guilleijo) [inline target/serializers.py]: You could do `num_current_targets = user.target_set.count()` instead

File: target/tests/tests_views.py [hunk 2]
```diff
@@ -56,6 +57,22 @@ def test_create_target(self):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self.assertEqual(response.data.get('properties').get('user'), self.test_active_user.id)
 
+    def test_create_more_targets_than_it_is_allowed(self):
+        initial_num_of_targets = Target.objects.filter(user=self.test_active_user).count()
+        for i in range(0, (settings.MAX_NUMBER_OF_TARGETS-initial_num_of_targets)):
```
Comment (guilleijo) [inline target/tests/tests_views.py]: Add spaces around the minus sign

File: target/tests/tests_views.py [hunk 3]
```diff
@@ -56,6 +57,22 @@ def test_create_target(self):
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
         self.assertEqual(response.data.get('properties').get('user'), self.test_active_user.id)
 
+    def test_create_more_targets_than_it_is_allowed(self):
+        initial_num_of_targets = Target.objects.filter(user=self.test_active_user).count()
```
Comment (guilleijo) [inline target/tests/tests_views.py]: Same as before, using `self.test_active_user.target_set`

File: target/tests/tests_views.py (hunk 2 above)
Comment (MathiasLantean) [reply target/tests/tests_views.py]: How flake8 didn't fail in this case. I totally agree.

**Review by guilleijo — APPROVED** [review]
👍

**Review by kaozdl — APPROVED** [review]

**Pull request merged**